inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -6,36 +6,36 @@
 import java.util.LinkedList;
 import java.util.StringJoiner;
 
-public class Restaurant {
+public class Restaurant extends Store {
 
-    LinkedList<Reviews> restaurantsReviews = new LinkedList<>();
     public String name;
     public int stars;
     public int priceCategory;
```

Comment:
Again, you don't need this `int priceCategory` here since it's already contained in the super class `Store`